NSrtm.Core/Pgm/Grid/IPgmGrid.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NSrtm.Core.Pgm.Grid
+{
+    public interface IPgmGrid
```

Comment:
Please define word "Grid" becouse i donw understand what this class is. For reference:
http://www.thefreedictionary.com/grid
https://www.google.com/search?q=grid&es_sm=93&biw=1920&bih=1099&source=lnms&tbm=isch&sa=X&ved=0CAYQ_AUoAWoVChMIi9Genc3JxwIVYyZyCh0tfwj-